website/docs/docs/polling-bdc.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Nice! Will it work if we use the same pattern as the open source, and put it in a "/go" directory? It would be ideal if the short URL could be `https://docs.pactflow.io/go/cid-polling-bdc`

Reply:
yes It should be possible, the routing is based on the folder structure. Let me try it out